### DISCUSSION TITLE
custom sound effects for notifications

### DISCUSSION THREAD

**Discussion by pinkjimrussiagirl** [stoatchat/discussions#616] [Feature Suggestions]
i don’t think anyone has requested this yet, but please add an option for people to add their own sounds as notifications if they want to use their own notification sounds

**Comment by Rouwrf** [comment]
Heavily agree with this. It would be incredibly useful if custom notification sounds could also be assigned to servers or specific contacts. At the very least, separate chimes for server notices and contacts by default would be a godsend considering how it was always an annoyance to think that one of your friends messaged you when a server would @ everyone.

**Comment by BalaDeSilver** [comment]
Are there any notification sounds at all on desktop???

I have the Linux app, and the webapp, and neither do notification sounds...

**Reply by Cosmo987874783** [reply]
there are not

**Comment by Cosmo987874783** [comment]
+1 to this, because I would be fine with there just being normal notification sounds but there being built in ways to add your own sounds would be extremely nice

**Comment by PyroNicampt** [comment]
This would help me a lot, as system notifications are entirely muted for me and I have to manually check if I have a message
